src/common/redux/states/import.ts

```diff
@@ -6,6 +6,13 @@
 // ==LICENSE-END==
 
 export interface ImportState {
-    publication: any;
+    publication: ImportOpdsPublication;
     downloadSample: boolean;
 }
+
+export interface ImportOpdsPublication {
+    url: string;
+    base64OpdsPublication: string;
+    downloadSample: boolean;
```

Comment:
The `downloadSample` boolean member is defined twice (in the `ImportState` and `ImportOpdsPublication` interfaces). It is indeed perfectly logical for `lastImport: ImportState` to convey the `downloadSample` true/false property. However I think we should remove the `downloadSample` member from the `ImportOpdsPublication` data structure, and change the signature of the `importOpdsEntry()` function accordingly:
`importOpdsEntry(data: ImportOpdsPublication, downloadSample: boolean): Promise<PublicationView>`

The alternative (which I personally do not like) would be to remove the `downloadSample` member from the `ImportOpdsPublication` data structure, and to create an extended type (interface) specifically for the parameter of the `importOpdsEntry()` function:
```
export interface ImportOpdsPublicationWithDownloadSample extends ImportOpdsPublication {
    downloadSample: boolean;
}
```